src/components/space/charts/ActionsByUserChart.js

```diff
@@ -0,0 +1,112 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import {
+  BarChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Bar,
+  ResponsiveContainer,
+} from 'recharts';
+import EmptyChart from './EmptyChart';
+import {
+  getActionNumByVerb,
+  filterActionsByUser,
+  findYAxisMax,
+} from '../../../utils/api';
+import { COLORS, CONTAINER_HEIGHT } from '../../../config/constants';
+import { DataContext } from '../../context/DataProvider';
+
+const ACTION_TYPES = [
+  'get',
+  'get_children',
+  'update',
+  'create',
+  'delete',
+  'copy',
+  'move',
+];
+
+const useStyles = makeStyles(() => ({
+  typography: { textAlign: 'center' },
+}));
+
+const ActionsByUserChart = () => {
+  const maxUsers = 10;
+  const title = 'Actions by Most Active Users';
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const { actions, selectedUsers, allMembers } = useContext(DataContext);
+  let users;
+  if (
+    selectedUsers === null ||
+    selectedUsers.length === 0 ||
+    selectedUsers.length === allMembers.length
+  ) {
+    users = allMembers;
+  } else {
+    users = selectedUsers;
+  }
+  const yAxisMax = findYAxisMax(users);
+  let formattedActions = [];
+
+  users.forEach((user) => {
+    const actionsByVerb = getActionNumByVerb(
+      filterActionsByUser(actions, [{ id: user.id }]),
+    );
+    const userActions = {
+      id: user.id,
+      name: user.name,
+      total: actionsByVerb.total,
+    };
+    ACTION_TYPES.forEach((actionType) => {
+      if (actionsByVerb[actionType]) {
+        userActions[actionType] = actionsByVerb[actionType];
+      }
+    });
```

Comment:
See my comment below, but it would be better to create the object with all the action types instead of only the selected ones.

Reply:
I used to count all the types, but then I think there would be too much irrelevant information
![image](https://user-images.githubusercontent.com/44131581/176690364-fc142e06-3145-414b-a511-8b52f30ecfb6.png)

Reply:
Then maybe hide the zero ones? Or maybe sort them. I think we'll have an issue later when we increase the different types, but let's see later ^^'

Reply:
If we hide zeros or sort them, the action and the color of the bar will have different orders, then we lose part of the benefits of stacked bars.

Reply:
I see, but let's count all the types anyway, later we will let the user filter out data if he wants to.